install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -4,22 +4,22 @@
 
 global:
   # The Istio control plane Pilot endpoint
-  pilotEndpoint: istio-pilot.istio-system
+  pilotEndpoint:
```

Comment:
seems like a little too much overloading.  I believe @costinm had thought using the actual DNS name would result in a workable solution with virtual services + coredns in addition.

One thing that might work a little better is:
```
global.pilotEnabled: true
global.telemetryEnabled: true
global.tracingEnabled: false
```

and not overloading the variables.

Reply:
I think a point is that having default values don't make sense anyway.  The endpoint manifest won't validate with those defaults--the API demands an IP for the field.

Reply:
@sdake adding a new parameter to control the endpoint is enabled or not seems to be complex, as I need to update two places if I want to enable pilotEndpoint, including both `global.pilotEnabled` and `pilotEndpoint`, but with this PR, we only need to update `pilotEndpoint`.